### ISSUE TITLE
Feature request: configurable position and drop down list size

### ISSUE THREAD

**Issue by Boatsandlogs** [sweetgiorni/AnyPortal#8]
Hi, im not sure where the feature request page is so I am opening an issue

I would like to put the drop down list at the top of the screen and allow it to go all the way to the bottom of the screen because I make a TON of portals.  can we define  appear position 1200x,100y, height=1200px width = 600px for example?

**Comment by sweetgiorni** [comment]
How about a search bar instead?

**Comment by Boatsandlogs** [comment]
that sounds pretty good, is the other one a lot of work

**Comment by sweetgiorni** [comment]
No, I just don't like it. Moving the dropdown to the top of the screen would be confusing and weird looking. Also, if someone really has a lot of portals they'd still end up scrolling anyway, only with a slightly larger scrollbar. It doesn't really remedy the problem; it just makes a big ugly list take up the entire height of your screen.
